building/README.md

```diff
@@ -138,20 +239,71 @@ You can read more about the building script options [here](script.md).
 This is the method preferred when you plan to develop Phoenix-RTOS.
 
 Firstly, you need to install some tools required for compiling the toolchain and finally create a Phoenix-RTOS system image.
-There is a list of commands you can use to get them on the Ubuntu 20.04 host operating system.
+There is a list of commands you can use to get them: on both Ubuntu and MacOS host operating systems.
+
+  <details>
+  <summary>Intalling required tools for native build on Ubuntu (click to expand)</summary>
+
+  ```bash
+  sudo apt-get update && \
+  sudo apt-get upgrade && \
+  sudo apt-get install build-essential \
+  mtd-utils \
+  autoconf \
+  texinfo \
+  genext2fs \
+  libtool \
+  libhidapi-dev \
+  python3
+  ```
+
+  </details>
+
+  <details>
+  <summary>Intalling required tools for native build on MacOS (click to expand)</summary>
+
+  ```bash
+  brew update && \
+  brew upgrade && \
+  brew install bash \
```

Comment:
nit: maybe is worth to mention that we need bash in version higher than X - because bash is preinstalled in macos but in the older version. It also applies to `make`

Reply:
Do you remember what versions are required?

Reply:
I think for bash it's 5.+ and for make 4.+, but not 100% sure

Reply:
I think it's not so crucial to place the exact version (it may change in the future), but I left a short comment about it.

Reply:
we need bash >=4.0 for associative arrays
we need make >=3.82 for `undefine` directive to work

Reply:
Thanks, updated.